observability/otel/otel_rn/otel-rn-3-2.adoc

```diff
@@ -0,0 +1,80 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="otel-rn-3-2"]
+= Release notes for {OTELName} 3.2
+:context: otel-rn-3-2
+
+toc::[]
+
+include::modules/otel-product-overview.adoc[leveloffset=+1]
+
+The {OTELName} is provided through the {OTELOperator}.
+
+////
+[id="otel-rn_3-2_cves"]
+== CVEs
+
+This release fixes link:https://access.redhat.com/security/cve/cve-202?-?????[CVE-202?-?????].
+////
+
+[id="otel-rn_3-2_technology-preview-features"]
+== Technology Preview features
+
+This update introduces the following Technology Preview features:
+
+* Host Metrics Receiver
+* OIDC Auth Extension
+* Kubernetes Cluster Receiver
+* Kubernetes Events Receiver
+* Kubernetes Objects Receiver
+* Load-Balancing Exporter
+* Kubelet Stats Receiver
+* Cumulative to Delta Processor
+* Forward Connector
+* Journald Receiver
+* Filelog Receiver
+* File Storage Extension
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="otel-rn_3-2_new-features-and-enhancements"]
+== New features and enhancements
+
+This update introduces the following enhancement:
+
+* {OTELName} 3.2 is based on the open source link:https://opentelemetry.io/[OpenTelemetry] release 0.99.0.
```

Comment:
We should document that leaf empty objects or null in the `OpenTelemetryCollector` e.g.;:
```yaml
exporters:
  debug:
```

or
```yaml
exporters:
  debug: null
```

should be changed to an actual empty object

```yaml
exporters:
  debug: {}
```

PR with the change for our docs https://github.com/openshift/openshift-docs/pull/76135/files#diff-294ec585891d4f53f51203c945436edac35259850eb19f99f502661759203d21R846

Reply:
I added it as a deprecation notice.